.github/paths-labeller.yml

```diff
@@ -0,0 +1,3 @@
+---
+- 'team:observablt-ci':
+    - '.github/workflows/**/*.*'
```

Comment:
Let's add the Elastic Agent control plane team as the owners of all the other paths.